apps/ml/example_data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ActivityTracking": {
+    "accelerometer": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }],
+    "gyroscope": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }],
+    "magnetometer": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }]
+  }
+}
```

Comment:
Yeah, this should be typed with pydantic as I mentioned in comment above.

https://fastapi.tiangolo.com/tutorial/response-model/

https://christophergs.com/tutorials/ultimate-fastapi-tutorial-pt-4-pydantic-schemas/